docs/LwM2M_Client/STMicroelectronics/STM32L496G-DISCOBG96.md

```diff
@@ -46,8 +48,8 @@ Enter the command line interface on your machine and paste the following command
 0. In the Project Explorer, navigate to the **Anjay-freertos-client-STM32L496G-BG96** project:
     - Right-click on the project name and select **Build Project**. The build should take less than one minute to complete.
     - After the build is finished, right-click on the project name, select **Run As** and click the **1 STM32 Cortex-M C/C++ Application** option.
-        - In the **Lauch Configuration Selection**, choose the **Anjay-freertos-client-STM32L496G-BG96** option and click **OK**.
-0. After the build and run are complete, the board is now compiled.
+    - In the **Lauch Configuration Selection**, choose the **Anjay-freertos-client-STM32L496G-BG96** option and click **OK**.
```

Comment:
I can't add a comment in the line `47`, but the "Import project" image should be updated.